aten/src/ATen/native/cuda/BinaryMulDivKernel.cu

```diff
@@ -9,6 +9,39 @@
 
 namespace at { namespace native {
 
+template<typename scalar_t>
+struct InvDivFunctor {
+    InvDivFunctor(scalar_t inv): inv_b(inv) {}
+    __device__ scalar_t operator() (scalar_t a) const {
+      return a * inv_b;
+    }
+  private:
+    scalar_t inv_b;
+};
+
+template<typename scalar_t>
+struct DivFunctor {
+  __device__ scalar_t operator() (scalar_t a, scalar_t b) const {
+    return a / b;
+  }
+};
+
+template<typename scalar_t>
+struct MulFunctor {
+  __device__ scalar_t operator() (scalar_t a, scalar_t b) const {
+    return a * b;
+  }
+};
+
+// Workaround for the error: '*' in boolean context, suggest '&&' instead [-Werror=int-in-bool-context]
```

Comment:
lol it does not complain in AddFunctor

Reply:
This looks like a host only check, so if lambdas no longer has to be host/device then it can probably be replace with '*'